website/content/docs/release-notes/1.14.0.mdx

```diff
@@ -21,6 +21,8 @@ official guidance until the release has been completed.
 
 @include 'ui-pki-control-groups-known-issue.mdx'
 
+@include 'update-primary-known-issue.mdx'
```

Comment:
Could you please add to 1.13 release notes & upgrade guide as well?

Reply:
Yeah, I was wondering how far back we should go.  Just 1.13 then?